CODE_OF_CONDUCT.md

```diff
@@ -29,6 +29,18 @@ Project maintainers are responsible for clarifying the standards of acceptable b
 
 Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
 
+## Disclaimer
+
+The MSTG and it's materials are for educational and research purposes only, or if you are a professional security tester that is analysing mobile apps with approval of the app owner (e.g. Penetration Test or Bug Bounty Program).
+
+Please consult the laws in your country before executing any tests against mobile apps by utilizing the MSTG materials. Refrain from violating the laws with anything described in the MSTG.
+
+The MSTG project is also NOT supporting in any way of unethical activities and if you come across such requests please reach out to the project leaders and raise this to us.
+
+Neither OWASP, project leaders, authors or anyone else involved in this project is going to accept responsibility for your actions.
```

Comment:
```suggestion
Neither OWASP, the MSTG project leaders, authors or anyone else involved in this project is going to accept responsibility for your actions.
```

Reply:
Apart from that: Accept or take responsibility?

Reply:
I made it now "take responsibility". Have a look